_non-cosmos/ethereum/assetlist.json

```diff
@@ -1201,6 +1201,34 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/ox.svg"
         }
       ]
+    },
+    {
+      "description": "BEAST: Battle, Breed, and Customize unique creatures powered by Ethereum & Cosmos",
```

Comment:
Can I recommend an `"extended_description": "A monster breeding & battling game with a twist. Each game interaction burns $BEAST tokens, adding a strategic depth to breeding, battling, and customization. Dive into a world where every move counts!"`
This is the description from Etherscan